__tests__/commands/install/integration.js

```diff
@@ -236,6 +236,13 @@ test.concurrent('install file: protocol without cache', async (): Promise<void>
   });
 });
 
+test.concurrent('install file: local packages with local dependencies', async (): Promise<void> => {
+  await runInstall({}, 'install-file-local-dependency', async (config, reporter) => {
+    const reinstall = new Install({}, config, reporter, await Lockfile.fromDirectory(config.cwd));
+    await reinstall.init();
```

Comment:
It is either assert that exception not thrown or assert that module has been installed in node modules